core-bundle/contao/templates/twig/form_checkbox.html.twig

```diff
@@ -34,7 +34,7 @@
                         .set('value', option.value)
                         .set('checked', option.checked, option.checked)
                         .mergeWith(option.attributes)
-                    }}> <label id="lbl_{{ option.id }}" for="opt_{{ option.id }}">{{ option.label }}</label></span>
+                    }}> <label id="lbl_{{ option.id }}" for="opt_{{ option.id }}">{{ option.label|insert_tag_raw|raw }}</label></span>
```

Comment:
What about

```suggestion
                    }}> <label id="lbl_{{ option.id }}" for="opt_{{ option.id }}">{{ option.label|sanitize_html('contao')|insert_tag_raw }}</label></span>
```

instead? This would cover both the raw HTML as well as the usage of insert tags as mentioned in https://github.com/contao/contao/pull/9096#pullrequestreview-3597884172.

Reply:
That work _by accident_ for my current use case, since I can control what tags to allow and what I use in my translation files. However, I actually don't want to allow users to enter HTML in the back end, I only want to use HTML in my DCA fields. 🙃

Reply:
> I actually don't want to allow users to enter HTML in the back end, I only want to use HTML in my DCA fields.

Use custom template in your project then. But do not change the core, as these template are meant to be used by the form generator.